docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -16,27 +17,81 @@ Proposed
+the light client isn’t expired.
 
-Current limitations:
+After having successfully verified a misbehaviour, the endpoint will execute the jailing and slashing of the malicious validators similarly as in the evidence module. 
+
+### Current limitations:
 
 - This only handles light client attacks, not double signing. In the future, we will add the code to also verify double signing.
-- Since a special endpoint must be used to submit the evidence, the evidence is not automatically submitted by Hermes. In the future, we may make Hermes submit the evidence, or use a hook provided by IBC to run the code automatically when evidence is submitted to the IBC client. In the current state, someone will need to submit the evidence manually in the 3 weeks after a light client attack has occured.
+
 - We cannot derive an infraction height from the evidence, so it is only possible to tombstone validators, not actually slash them.
-  To explain the technical reasons behind this limitation, let's recap the initial [consumer initiated slashing logic](https://github.com/cosmos/ibc/blob/2aa283d4bc8b747cbb860c0f94389d7a62bb802b/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#consumer-initiated-slashing). 
-  In a nutshell, consumer heights are mapped to provider heights through `VSCPackets`, namely through the so called `vscIDs`. 
-  When an infraction occurs on the consumer, a `SlashPacket` containing the `vscID` obtained from mapping the consumer infraction height is sent to the provider. 
-  Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height, which is used to slash the misbehaving validator. 
-  In the context of untrusted consumer chains, all their states, including `vscIDs`, could be corrupted and therefore cannot be used for slashing purposes.
+To explain the technical reasons behind this limitation, let's recap the initial consumer initiated slashing logic.
+In a nutshell, consumer heights are mapped to provider heights through VSCPackets, namely through the so called vscIDs.
+When an infraction occurs on the consumer, a SlashPacket containing the vscID obtained from mapping the consumer infraction height
+is sent to the provider. Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height,
+which is used to slash the misbehaving validator. In the context of untrusted consumer chains, all their states, including vscIDs,
+could be corrupted and the ore cannot be used for slashing purposes.
+
+- Currently, the endpoint can only handle "equivocation" light client attacks. This is because the "lunatic" attacks require the endpoint to possess the ability to dissociate which header is conflicted or trusted upon receiving a misbehavior message. Without this information, it's not possible to define the Byzantine validators from the conflicting headers (see [comment](https://github.com/cosmos/interchain-security/pull/826#discussion_r1268668684)).
```

Comment:
I don't think I understand this, maybe you can clarify. What does "the ability to dissociate which header is conflicted or trusted" mean? For the provider, none of the headers can be trusted and this is why the client will be frozen (assuming headers verify). By looking at the headers we can determine the validator(s) that signed both and these should be punished. This should cover all light client attacks.

Reply:
> By looking at the headers we can determine the validator(s) that signed both and these should be punished. This should cover all light client attacks.

This is true for equivocation, but not for lunatic. Sure, even for lunatic you end up jailing the intersection, but some malicious validators will not be punished.

Reply:
but it's the best we can do, no?

Reply:
For now yes, but we should be able to jail for lunatic attacks as well. It may be the case that the intersection for a lunatic attack is an empty set.